docs/docusaurus.config.js

```diff
@@ -39,7 +39,15 @@ const codeTheme = require('./src/utils/prism');
     themeConfig:
       /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
       ({
-        image: '/img/social-preview.jpg',
+        metadata: [
+          { name: 'robots', content: 'max-image-preview:large' },
+          {
+            name: 'keywords',
+            content:
+              'novu,novu documentation,novu docs,notification,notification insfrastructure,open source,oss',
+          },
+        ],
+        image: 'https://novu.co/images/social-preview.jpg',
```

Comment:
```suggestion
        image: '/img/social-preview.jpg',
```

this asset is also available on the docs project, any specific reason you wanted. to fetch it form the landing page?

Reply:
right now it is not working, showing 404 😅

Reply:
If we share doc link, image is not showing because this asset is missing

Reply:
Thanks Andrew, I missed this